source/macos/AdaptiveCards/AdaptiveCards/AdaptiveCard.swift

```diff
@@ -110,3 +114,19 @@ public struct HyperlinkColorConfig {
         self.underlineStyle = underlineStyle
     }
 }
+
+public struct ChoiceSetButtonConfig {
+    public static let `default` = ChoiceSetButtonConfig(onIcon: nil, offIcon: nil, onHoverIcon: nil, offHoverIcon: nil)
+    
+    let onIcon: NSImage?
+    let offIcon: NSImage?
+    let onHoverIcon: NSImage?
+    let offHoverIcon: NSImage?
```

Comment:
```suggestion
    let onIcon: NSImage
    let offIcon: NSImage
    let onHoverIcon: NSImage
    let offHoverIcon: NSImage
```
These should be non-optional.
If custom images are provided, then we need all of these variants to show.